File: B/C++/DataTim.cpp
#include <string>
#include <iostream>

using namespace std;

// Kelas DataTim
class DataTim
{
    // Atribut Private
    private:
        string tim;
        string negara;
        int tahun;
        string pemain;

    // Atribut Public
    public:
        // Constructor
        DataTim()
        {
        
        }

        // Constructor Input
        DataTim(string tim, string negara, int tahun, string pemain)
        {
            this->tim = tim;
            this->negara = negara;
            this->tahun = tahun;
            this->pemain = pemain;
        }

        // Prosedur Setter
        void setTim(string tim)
        {
            this->tim = tim;
        }

        void setNegara(string negara)
        {
            this->negara = negara;
        }

        void setTahun(int tahun)
        {
            this->tahun = tahun;
        }

        void setPemain(string pemain)
        {
            this->pemain = pemain;
        }

        // Fungsi Getter
        string getTim()
        {
            return this->tim;
        }

        string getNegara()
        {
            return this->negara;
        }       

        int getTahun()
        {
            return this->tahun;
        }

        string getPemain()
        {
            return this->pemain;
        }

        // Destructor
        ~DataTim()
        {

        };

        // Prosedur Print Data
        void printData()
        {
            cout << "Tim            : " << this->getTim() << endl;
            cout << "Negara         : " << this->getNegara() << endl;
            cout << "Tahun Berdiri  : " << this->getTahun() << endl;
            cout << "Pemain         : " << this->getPemain() << endl;
        }
};

File: B/Java/DataTim.java
// Kelas DataTim
class DataTim
{
    // Atribut Private
    private String tim;
    private String negara;
    private int tahun;
    private String pemain;
    
    // Constructor
    DataTim()
    {

    }

    // Constructor Input
    public DataTim(String tim, String negara, int tahun, String pemain)
    {
        this.tim = tim;
        this.negara = negara;
        this.tahun = tahun;
        this.pemain = pemain;
    }

    // Getter dan Setter Tim
    public String getTim() {
        return tim;
    }

    public void setTim(String tim) {
        this.tim = tim;
    }

    // Getter dan Setter Negara
    public String getNegara() {
        return negara;
    }

    public void setNegara(String negara) {
        this.negara = negara;
    }

    // Getter dan Setter Tahun
    public int getTahun() {
        return tahun;
    }

    public void setTahun(int tahun) {
        this.tahun = tahun;
    }

    // Getter dan Setter Pemain
    public String getPemain() {
        return pemain;
    }

    public void setPemain(String pemain) {
        this.pemain = pemain;
    }
    
    // Print Data
    public void printData()
    {
        System.out.println("Tim             : " + this.getTim());
        System.out.println("Negara          : " + this.getNegara());
        System.out.println("Tahun Berdiri   : " + this.getTahun());
        System.out.println("Pemain          : " + this.getPemain());
    }
}
